x-pack/plugins/stack_alerts/public/alert_types/components/data_view_select_popover.tsx

```diff
@@ -0,0 +1,90 @@
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useCallback, useEffect, useState } from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiExpression, EuiPopover } from '@elastic/eui';
+import { DataViewsList } from '@kbn/unified-search-plugin/public';
+import { DataViewListItem } from '@kbn/data-views-plugin/public';
+import { useTriggersAndActionsUiDeps } from '../es_query/util';
+
+interface DataViewSelectPopoverProps {
+  onSelectDataView: (newDataViewId: string) => void;
+  initialDataViewTitle: string;
+  initialDataViewId?: string;
+}
+
+export const DataViewSelectPopover: React.FunctionComponent<DataViewSelectPopoverProps> = ({
+  onSelectDataView,
+  initialDataViewTitle,
+  initialDataViewId,
+}) => {
+  const { data } = useTriggersAndActionsUiDeps();
+  const [dataViewItems, setDataViewsItems] = useState<DataViewListItem[]>();
+  const [dataViewPopoverOpen, setDataViewPopoverOpen] = useState(false);
+
+  const [selectedDataViewId, setSelectedDataViewId] = useState(initialDataViewId);
+  const [selectedTitle, setSelectedTitle] = useState<string>(initialDataViewTitle);
+
+  useEffect(() => {
+    const initDataViews = async () => {
+      const fetchedDataViewItems = await data.dataViews.getIdsWithTitle();
+      setDataViewsItems(fetchedDataViewItems);
+    };
+    initDataViews();
+  }, [data.dataViews]);
+
+  const closeDataViewPopover = useCallback(() => setDataViewPopoverOpen(false), []);
+
+  if (!dataViewItems) {
+    return null;
+  }
+
+  return (
+    <EuiPopover
+      id="dataViewPopover"
+      button={
+        <EuiExpression
+          display="columns"
+          data-test-subj="selectDataViewExpression"
+          description={i18n.translate('xpack.stackAlerts.components.ui.alertParams.dataViewLabel', {
+            defaultMessage: 'data view',
+          })}
+          value={selectedTitle}
+          isActive={dataViewPopoverOpen}
+          onClick={() => {
+            setDataViewPopoverOpen(true);
+          }}
+          isInvalid={!selectedTitle}
+        />
+      }
+      isOpen={dataViewPopoverOpen}
+      closePopover={closeDataViewPopover}
+      ownFocus
+      anchorPosition="downLeft"
+      zIndex={8000}
+      display="block"
+    >
+      <div style={{ width: '450px' }} data-test-subj="chooseDataViewPopoverContent">
+        <DataViewsList
+          dataViewsList={dataViewItems}
+          onChangeDataView={(newId) => {
+            setSelectedDataViewId(newId);
+            const newTitle = dataViewItems?.find(({ id }) => id === newId)?.title;
+            if (newTitle) {
+              setSelectedTitle(newTitle);
+            }
+
+            onSelectDataView(newId);
+            closeDataViewPopover();
+          }}
+          currentDataViewId={selectedDataViewId}
+        />
+      </div>
+    </EuiPopover>
```

Comment:
I think for consistency this should also work like the other fields with a title and and x to close it 
![Bildschirmfoto 2022-05-18 um 10 14 16](https://user-images.githubusercontent.com/463851/168991548-4a968566-b327-4995-b2f5-b3180d79b610.png)

![Bildschirmfoto 2022-05-18 um 10 16 16](https://user-images.githubusercontent.com/463851/168991608-78ce66a3-41e0-4205-b011-4032b64697e9.png)

Reply:
Fixed